src/home/HomePage.test.tsx

```diff
@@ -34,6 +34,15 @@ describe('HomePage', () => {
     isLoading: false,
     isError: false,
   });
+
+  it('sets correct document title', async () => {
+    render(<HomePage />);
+
+    await waitFor(() => {
+      expect(document.title).toBe(process.env.SITE_NAME);
```

Comment:
[question] Should this title contain `Catalog` or more relevant for this page text?

Reply:
Yes, except for "|" we have moved the name from legacy